site/phpstan-baseline.neon

```diff
@@ -13619,3 +13619,8 @@ parameters:
 			message: "#^Function exception_handler\\(\\) has no return type specified\\.$#"
 			count: 1
 			path: public/index.php
+
+		-
+			message: "#^Offset int\\<1, max\\> does not exist on non\\-empty\\-array\\<0, \\(float\\|int\\)\\>\\.$#"
+			count: 2
+			path: app/controllers/forum/ForumController.php
```

Comment:
How hard would it be to fix this error rather than ignoring it?  If you need help fixing it, message me on Zulip or come find me during a meeting.